### PR TITLE
Add the french translation to RPChat

### PR DESCRIPTION
**Pull request by Oldarorn**

### CODE REVIEW

File: locates/fr.lua [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Locales['fr'] = {
+	-- ooc chat
+	['ooc_help'] = 'Écrit un message hors du personnage',
```
Comment (ElPumpo) [inline locates/fr.lua]: All translations start lower case

**Comment by nearbyplayer** [comment]
You put “HRP |” you’ll need to change that because your server initials shouldn’t be public.

File: locates/fr.lua [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	['ooc_prefix'] = 'HRP | %s',
```
Comment (ElPumpo) [inline locates/fr.lua]: Change to something generic and not server related

Comment (Oldarorn) [reply locates/fr.lua]: HRP means Hors RolePlay (out of...) :) I'll fix it !

**Comment by Oldarorn** [comment]
It should be good now !

**Comment by ElPumpo** [comment]
Merged, thanks for your help.

**Pull request merged**